### DISCUSSION TITLE
latenite/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#397] [Website Discussions]
# latenite/

This website has been thrown together by Chris Hocking at LateNite.

https://fcp.cafe/latenite/

**Comment by skymakai** [comment]
I'll be looking out for Magic Toolbox!

**Reply by latenitefilms** [reply]
Sadly Magic Toolbox is still in the bottom drawer, but I know there'll be some amazing stuff announced around the time of the FCP Creative Summit!
